src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" 
+      data-qa="card"
+    >
+      <img class="card__image" src="./images/imac.jpeg" alt="imac">
+
+      <h3 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <div class="card__product-code">
+        Product code: 195434
+      </div>
+      <div class="stars card__stars">
+        <div class="stars__star stars__star--4">
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+        </div>
+    
+        <div class="reviews">
+          Reviews: 5
+        </div>
+      </div>
+      
+      <div class="price">
+        <div class="price__text">Price:</div>
+        <div class="price__money">$2,199</div>
+      </div>
+  
+      <div class="button">
+        <a class="button__link" 
+          href="#buy" 
+          data-qa="hover"
+        >
```

Comment:
```suggestion
        <a
          class="button__link" 
          href="#buy" 
          data-qa="hover"
        >
```
pay attention to the code style, fix it all over the page